life_expectancy/tests/test_cleaning.py

```diff
@@ -1,15 +1,16 @@
 """Tests for the cleaning module"""
 import pandas as pd
 
-from life_expectancy.cleaning import DATA_DIR, FILE_NAME, FILE_PATH, main
+from life_expectancy.main import FILE_NAME, main
 
-from . import OUTPUT_DIR
+from . import FIXTURES_DIR, OUTPUT_DIR
 
 
 def test_clean_data(pt_life_expectancy_expected):
     """Run the `clean_data` function and compare the output to the expected output"""
-    main(FILE_PATH, FILE_NAME, DATA_DIR)
-    pt_life_expectancy_actual = pd.read_csv(OUTPUT_DIR / "pt_life_expectancy.csv")
+    eu_life_expectancy_actual = main(
```

Comment:
Here you are still saving test data to the file system. You should add a patch here as well so that no data is actually saved